app/extensions/FlyToGeoJsonExtension.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Layer,
+  LayerExtension,
+  UpdateParameters,
+  FlyToInterpolator,
+  WebMercatorViewport,
+} from '@deck.gl/core';
+import { bbox } from '@turf/bbox';
+import { MultiPolygon, MultiPoint } from '~/gen';
+
+export class FlyToGeoJsonExtension extends LayerExtension {
+  updateState(
+    this: Layer,
+    { props, oldProps, changeFlags }: UpdateParameters<Layer>,
+  ): void {
+    const { deck: deckInstance } = this.context;
+    if (deckInstance === undefined) {
+      throw new Error('Deck instance undefined in FlyToGeoJsonExtension');
+    }
+    if (
+      !Array.isArray(props.data) &&
+      ((Array.isArray(oldProps.data) && oldProps.data.length === 0) ||
+        changeFlags.dataChanged)
+    ) {
+      const viewport = this.context.viewport as WebMercatorViewport;
+      const [minX, minY, maxX, maxY] = bbox(
+        props.data as MultiPolygon | MultiPoint,
```

Comment:
If we swap to `@types/geojson`, we can use the [Geometry type](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/865051c6befe8d67f7ac2fdcac6cdf4922c1dc83/types/geojson/index.d.ts#L69), which supports all the shapes- not just the two defined in our OpenAPI spec.